### DISCUSSION TITLE
Firebase data connection setup

### DISCUSSION THREAD

**Discussion by Flickerko** [Budibase/budibase#6228] [Help] [answered]
I'm attempting to setup connection to Firebase database unsuccessfully. Does anyone have a guide on how to do it? I've tried following this link, got up to the point of creating service account and reading private key, but not sure how to fill it out:
https://cloud.google.com/docs/authentication/getting-started

Screenshot of how I filled out access details attached.

![image](https://user-images.githubusercontent.com/44823354/172448769-fa02b201-25dd-4843-967a-c78bf80e07ac.png)

![image](https://user-images.githubusercontent.com/44823354/172448832-6ec07052-81da-46c1-a3ab-bac8a31f12c0.png)

**Comment by melohagan** [comment, marked as answer]
Hi @Flickerko 

Firstly, you do not need the ServiceAccount field. Please remove that value.

Secondly, there was a bug with the private key parsing. This has been fixed recently, and will be released with the next hotfix build: https://github.com/Budibase/budibase/pull/6269

Docs have also just been added: https://docs.budibase.com/docs/firestore
